commit 6972fe96e8ba69f5ef25920784e7bba13f5b83ce
Author: maximilian attems <maks@sternwelten.at>
Date:   Sun Jul 23 22:13:37 2006 +0200

    Revert "[klibc] Use vfork() rather than fork(), to be nice to uClinux"
    
    hppa has no vfork,
    uClinux not yet relevant for Debian klibc.
    
    This reverts 044f2b0daa4768e0cdc0af5775fc215ec2c11bd8 commit.

diff --git a/usr/kinit/initrd.c b/usr/kinit/initrd.c
index 3827cd6..1371333 100644
--- a/usr/kinit/initrd.c
+++ b/usr/kinit/initrd.c
@@ -115,7 +115,7 @@ static int run_linuxrc(int argc, char *a
 	    mount(".", "/", NULL, MS_MOVE, NULL) || chroot("."))
 		return -errno;
 
-	pid = vfork();
+	pid = fork();
 	if (pid == 0) {
 		setsid();
 		/* Looks like linuxrc doesn't get the init environment
